package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "barrelsby",
-  "version": "0.0.4",
```

Comment:
Don't need to resolve this (as I can do it when I release) but there is already a `v0.0.5` I released a beta for #34 so this will be released as `v0.0.6`.